ci-operator/config/redhat-et/microshift/redhat-et-microshift-main.yaml

```diff
@@ -49,16 +56,35 @@ images:
       - destination_dir: .
         source_path: /go/src/github.com/redhat-et/microshift/_output/bin/linux_arm64/microshift
   to: linux-arm64-4.7.0-0.microshift
-promotion:
-  name: microshift
-  namespace: microshift
+- dockerfile_literal: "FROM os\nCOPY mock-nss.sh /bin/mock-nss.sh\nCOPY google-cloud-sdk.repo
+    /etc/yum.repos.d/google-cloud-sdk.repo\nCOPY microshift /usr/bin/microshift\nRUN
+    yum update -y && \\\n  yum install --setopt=tsflags=nodocs -y \\\n  glib2-devel
+    \\\n  glibc-devel \\\n  glibc-static \\\n  gpgme-devel \\\n  libassuan-devel \\\n
+    \ genisoimage \\\n  gettext \\\n  google-cloud-sdk \\\n  nss_wrapper \\\n  openssh-clients
+    && \\\n  yum clean all && rm -rf /var/cache/yum/* \n"
+  from: os
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/src/github.com/redhat-et/microshift/_output/bin/linux_amd64/microshift
+    libvirt-installer:
+      paths:
+      - destination_dir: .
+        source_path: /bin/mock-nss.sh
+      - destination_dir: .
+        source_path: /etc/yum.repos.d/google-cloud-sdk.repo
+  to: microshift-setup
 resources:
   '*':
     limits:
       memory: 4Gi
     requests:
       cpu: 100m
       memory: 200Mi
+tag_specification:
```

Comment:
This I thought declared the microshift images as a component of OCP, where as promotion would isolate it from OCP releases.  Does it matter here which we use?